File: 1-Iniciante/Calculo-Simples/1010.c
#include <stdio.h>

int main() {

    int cod, qnt;
    float num;
    float valor[4] = {0, 0, 0, 0};
    for (int i = 0; i < 2; i++) {
    scanf ("%d %d %f", &cod, &qnt, &num);
    if (valor[0] == 0.0 && valor[1] == 0.0) {
        valor[0] = qnt;
        valor[1] = num;
    }
    else {
        valor[2] = qnt;
        valor[3] = num;
    }
    }
    printf ("VALOR A PAGAR: R$ %.2f\n", valor[0] * valor[1] + valor[2] * valor[3]);
    
    return 0;
}

File: 1-Iniciante/1010-Calculo-Simples/1010.py
valor = []
for _ in range (2):
    cod, qt, num = input().split()
    valor.append(int(qt))
    valor.append(float(num))
print ("VALOR A PAGAR: R$ {:.2f}" .format(valor[0] * valor[1] + valor[-2] * valor[-1]))